source/Outfit.cpp

```diff
@@ -118,17 +118,27 @@ const vector<string> Outfit::CATEGORIES = {
 
 void Outfit::Load(const DataNode &node)
 {
-	if(node.Size() >= 2)
+	if(node.Size() == 2)
 	{
 		name = node.Token(1);
 		pluralName = name + 's';
 	}
+	else if(node.Size() >= 3)
+	{
+		name = node.Token(2);
+		parentName = node.Token(1);
```

Comment:
One review comment: if tracking this, use a `GameData::Outfits().Get` to store the pointer, rather than a string that is later used to get the pointer.